mycore-base/src/main/java/org/mycore/datamodel/common/ProjectDirState.java

```diff
@@ -0,0 +1,7 @@
+package org.mycore.datamodel.common;
+
+public enum ProjectDirState {
```

Comment:
Is this enum still needed? Or can it be simplified to a boolean?

Reply:
The enum was proposed by someone else and i think this solution has both advantages included:
* We have a (minimalistic) stable boolean wrapper function
* The Enum can be freely extended without changing any current code dependent on it
(Or at least minimal&obvious changes if the inequality should go in another direction longer term for whatever reason)
=> to advertise it a bit: maximum flexibility with minimalistic (but still guiding) implementation